app/templates/more-info.hbs

```diff
@@ -11,47 +11,54 @@
         <div class="required field">
-          <h4 class="ui dividing header">License</h4>
+          <label>Frequency of update</label>
+          {{#ui-dropdown class="selection frequency" selected=model.update_frequency onChange=(action "setFrequency") as | execute mapper|}}
+            <div class="default text">Frequency</div>
+            <i class="dropdown icon"></i>
+            <div class="menu">
+              {{#each frequencies as | frequency |}}
+                <div class="item" data-value={{map-value mapper frequency}}>
+                  {{frequency}}
+                </div>
+              {{/each}}
+            </div>
+          {{/ui-dropdown}}
+        </div>
+        <div class="required field">
           <label>Does the data have a license?</label>
           <button class="ui compact button {{if (eq model.license_exists true) "active"}}" {{action "licenseExists" true}}>Yes</button>
           <button class="ui compact button none {{if (eq model.license_exists false) "active"}}" {{action "licenseExists" false}}>No</button>
           <button class="ui compact button none {{if (eq model.license_exists "unknown") "active"}}" {{action "licenseExists" "unknown"}}>I don't know</button>
         </div>
         {{#if (eq model.license_exists true)}}
           <form class="ui form">
-            <div class="inline required fields">
-              <label> Add a license <span class="required"> * </span> </label>
-              <div class="grouped fields">
-                <div class="field">
-                  {{ui-radio name="license" label="Choose from list of licenses" value="choose" current=licenseType onChange=(action (mut licenseType))}}
+            <div class="required grouped fields">
+              <label> Add a license </label>
+              <div class="fields">
+                <div class="eight wide field">
+                  {{ui-radio class="choose" label="Choose from license" value="choose" name="license" current=(if (eq model.user_submitted_url null) "choose")}}
+                  <div class="field">
+                    {{#ui-dropdown class="license selection" selected=model.license click=(action "setRadioButton" "choose") onChange=(action "selectLicense") as | execute mapper|}}
+                      <div class="default text">Licenses</div>
+                      <i class="dropdown icon"></i>
+                      <div class="menu">
+                        {{#each licenses as | license |}}
+                          <div class="item" data-value={{map-value mapper license}}>
+                            {{license.name}}
+                          </div>
+                        {{/each}}
+                      </div>
+                    {{/ui-dropdown}}
+                  </div>
                 </div>
-                <div class="field">
-                  {{ui-radio name="license" label="Provide link to license" value="provide" current=licenseType onChange=(action (mut licenseType))}}
```

Comment:
This PR didn't pick up  this  `licenseType`. (Which is needed to check errors) see diff with master.